packages/paste-design-tokens/contrast-rating/utils.ts

```diff
@@ -0,0 +1,204 @@
+import ColorCombos from 'color-combos';
+import type {ColorCombo} from 'color-combos';
+import type {GenericTokenShape, AllGenericTokens} from '../types/GenericThemeShape';
+import * as DefaultRawTokenJSON from '../dist/tokens.raw.json';
+import type {DesignToken, DesignTokensJSON, TokenPairContrastRating} from '../types';
+
+function toCamelCase(text: string): string {
+  return text.replace(/-\w/g, clearAndUpper);
+}
+
+function clearAndUpper(text: string): string {
+  return text.replace(/-/, '').toUpperCase();
+}
```

Comment:
Lodash has a camelCase function we could use here: https://lodash.com/docs/4.17.15#camelCase

Reply:
Nice catch, replaced